app/models/member.rb

```diff
@@ -32,6 +32,7 @@ class Member < ActiveRecord::Base
   scope :confirmed, -> { where('confirmed_at IS NOT NULL') }
   scope :located, -> { where("location <> '' and latitude IS NOT NULL and longitude IS NOT NULL") }
   scope :recently_signed_in, -> { reorder('updated_at DESC') }
+  scope :recently_joined, -> { reorder("created_at desc") }
```

Comment:
My concern is this.  Say that you have two members, Alice and Bob.

* Alice signs up.
* Bob signs up.
* Some time passes.
* Bob confirms.
* Now someone looks at the member page (or RSS feed, which should come next) and sees Bob's the most recently joined.
* More time passes
* Alice confirms.
* Now you look at the most recently signed up page/RSS feed and Bob is still the "most recent" and you might not notice Alice.  

Although Alice *signed up* first, she's not visible on the "most recent* page until she's confirmed, and she might not get noticed if she confirms out of order.

FWIW, I'm intending to subscribe to the RSS feed once we get it, so this is pretty relevant to me :)